src/ut_pex.cpp

```diff
@@ -685,9 +679,8 @@ namespace libtorrent
 		return boost::shared_ptr<torrent_plugin>(new ut_pex_plugin(*t));
 	}
 
-	bool was_introduced_by(peer_plugin const* pp, tcp::endpoint const& ep)
+	bool was_introduced_by(ut_pex_peer_store const* p, tcp::endpoint const& ep)
```

Comment:
none, I just wanted to hack the refactor with the minimal change. Does this comments mean that it's ok to add the extra member?